<launch>
    <node pkg="ekf" type="estimator_node" name="estimator_node" output="screen">
        <rosparam command="load" file="$(find ekf)/config/config.yaml" />
    </node>

    <node pkg="ekf" type="simulation_node" name="simulation_node" output="screen">
        <rosparam command="load" file="$(find ekf)/config/config.yaml" />
    </node>

    <node pkg="ekf" type="visualization_node" name="visualization_node" output="screen"/>

    <node pkg="ekf" type="measurement_node" name="measurement_node" output="screen">
        <rosparam command="load" file="$(find ekf)/config/config.yaml" />
    </node>

    <node pkg="ekf" type="controller_node" name="controller_node" output="screen">
        <rosparam command="load" file="$(find ekf)/config/config.yaml" />
    </node>

</launch>
